rules/macos/persistence_periodic_tasks_file_mdofiy.toml

```
@@ -28,7 +28,7 @@ type = "query"
 
 query = '''
 event.category:"file" and not event.type:"deletion" and
- file.path:(/private/etc/periodic/* or /private/etc/defaults/periodic.conf or /private/etc/periodic.conf)
+ file.path:("/private/etc/periodic/*" or "/private/etc/defaults/periodic.conf" or "/private/etc/periodic.conf")
```

this will treat the quoted `*` as literal

```suggestion
 file.path:(/private/etc/periodic/* or "/private/etc/defaults/periodic.conf" or "/private/etc/periodic.conf")
```